flow-plugins/flow-plugin-base/src/main/java/com/vaadin/flow/plugin/base/ConvertPolymerCommand.java

```diff
@@ -152,11 +156,31 @@ private List<Path> getFilePathsByGlob(Path baseDir, String glob)
         PathMatcher matcher = FileSystems.getDefault()
                 .getPathMatcher("glob:" + glob);
 
-        try (Stream<Path> walk = Files.walk(baseDir)) {
-            return walk.filter(path -> matcher.matches(path))
-                    .filter(path -> !path.toString().contains("node_modules"))
-                    .collect(Collectors.toList());
-        }
+        List<Path> matchingPaths = new ArrayList<>();
+
+        Files.walkFileTree(baseDir, new SimpleFileVisitor<Path>() {
+            @Override
+            public FileVisitResult preVisitDirectory(Path dir,
+                    BasicFileAttributes attrs) throws IOException {
```

Comment:
Here as well, maybe we could remove it:
```suggestion
                                                     BasicFileAttributes attrs) {
```